divvy/cloudfilebrowser.js

```diff
@@ -1015,10 +1025,14 @@ var cloudFileBrowser = (function() {
             var siteAddress = $(container).find('#site-address').val();
             if (element === 'sharepoint') {
             	siteAddress = $(container).find('#sp-site-address').val();
+            	apiKey = $(container).find('#client_key').val();
+            	apiSecret = $(container).find('#secret_key').val();
             }
             var callbackArgs = {
                 'element' : element,
-                'siteAddress': siteAddress
+                'siteAddress': siteAddress,
+                apiKey,
```

Comment:
Is this valid JS? I thought this bit of syntactic sugar was ES6 only... cc @letoss you know more JS than I do.

Reply:
@letoss  please let me know what you think, I personally think that this should just be 
callbackArgs = {element, siteAddress, apiKey}

Reply:
This is an ES6 syntax sugar as Gaby said. If this file belongs to the old FE, it may fail if it's run on IE11 or Edge.

Reply:
Reminder that we aren't supporting IE11 any more.